README.md

```diff
@@ -1,97 +1,7 @@
-3. pipenv install
-4. For Management only, rename the file `env_template` to `.env` and update the correct values inside with your
-   subscription ID and [Azure Service Principal credentials](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal).
-
-## Running the samples
-
-### Management Plane
-
-1. `create_eg_topics_and_event_subscriptions.py` creates a new event subscription for an Event Grid Topic, and send it to a webhook
-
-   In order for this sample to work, you need to have a valid `.env` file (see previous section).
-
-   **This sample requires a endpoint that supports EventGrid security validation. See https://aka.ms/esvalidation for details**
-
-   Run the sample : `pipenv run python create_eg_topics_and_event_subscriptions.py`
-
-2. `create_storage_event_subscriptions.py` creates a new event subscription for a storage account, filtering jpg creation, and send it to an Relay Hybrid Connection
-
-   In order for this sample to work, you need to have a valid `.env` file (see previous section).
-
-   Run the sample : `pipenv run python create_storage_event_subscriptions.py`
-
-3. `create_arm_event_subscriptions.py` creates a new event subscription for an Azure Subscription and send it to a Storage Queue
-
-   In order for this sample to work, you need to have a valid `.env` file (see previous section).
-
-   Run the sample : `pipenv run python create_arm_event_subscriptions.py`
-
-### Data plane
-
-1. `event_grid_publisher.py` publishes a custom event to a Topic. Topic might be created by the initial sample `create_eg_topics_and_event_subscriptions.py`
-   or [CLI](https://docs.microsoft.com/azure/event-grid/custom-event-quickstart#create-a-custom-topic).
-
-   In order for this sample to work, you need an Event Grid Topic and a Key (replace global variables in the script)
-
-   Run the sample : `pipenv run python event_grid_publisher.py`
-
-2. `ConsumerFunction` is an Azure Function app that receive an event from EventGrid. You can deploy the `ConsumerFunction` folder to an Azure Function using [VSCode](https://aka.ms/vscode-azure-functions),
-   this folder is ready to deploy. See the Readme in that directory for details.
-
-   If you just want the code, look at `ConsumerFunction/ConsumerFunction/run.py`.
-
-   Creating an Azure Function is out of the scope of this sample, you can follow [CLI tutorial](https://docs.microsoft.com/azure/azure-functions/functions-create-first-azure-function-azure-cli)
-   or [Portal tutorial](https://docs.microsoft.com/azure/azure-functions/functions-create-first-azure-function) or using [VSCode](https://aka.ms/vscode-azure-functions).
+The repository has been depreacted. Please follow the following links below for the samples related to the latest version of:
 
-## Resources
+* Management Plane: https://github.com/Azure-Samples/azure-samples-python-management/blob/master/samples/eventgrid/manage_servicename.py
```

Comment:
It might be a bit better pointing to the dir which contains a readme for instructions
```suggestion
* Management Plane: https://github.com/Azure-Samples/azure-samples-python-management/blob/master/samples/eventgrid
```